tests/TestCase/Core/Utility/FormTest.php

```diff
@@ -179,6 +199,29 @@ public function customControlOptionsProvider(): array
                     'type' => 'text',
                 ],
             ],
+            'lang options' => [
+                'lang',
+                [
+                    'type' => 'select',
+                    'options' => [
+                        [
+                            'value' => 'en',
+                            'text' => 'English',
+                        ],
+                        [
+                            'value' => 'de',
+                            'text' => 'Deutsh',
```

Comment:
minor... test works, but proper option text would be `'Deutsch'`

Reply:
Genau! :smiley_cat: